src/actions/authActions.ts

```diff
@@ -17,14 +20,15 @@ export const register = (user: IRegister) => async (dispatch: any) => {
     let res = await axios.post(`/api/account/register`, {
       method: "POST",
       headers: {
+        Accept: "application/json",
         "Content-Type": "application/json",
       },
       body: body,
     });
 
     console.log("Registration Payload" + JSON.stringify(body));
 
-    if (res.status === 200) {
+    if (res.status === 201) {
```

Comment:
is this not supposed to be a  `301` redirect code upon a successful registration?

Reply:
Ah I could be wrong  perhaps 301 would mean a redirect on failure, ill need to double-check this